sdk/storage/src/queue/requests/put_message_builder.json

```diff
@@ -4,12 +4,13 @@
   "uses": [
     "crate::prelude::*",
     "crate::responses::*",
-    "azure_core::errors::{check_status_extract_headers_and_body, AzureError}",
-    "azure_core::prelude::*",
+    "azure_sdk_core::errors::{check_status_extract_headers_and_body, AzureError}",
```

Comment:
The crate is now azure_core, no?

Reply:
Yes, my bad, I've overwritten the updates in the merge 🤦‍♂️ . 
I've pushed the correct imports now.